src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/config/AuditConfig.java

```diff
@@ -55,12 +140,42 @@ private Filter(final boolean isRestApiAuditEnabled,
             this.logRequestBody = logRequestBody;
             this.resolveIndices = resolveIndices;
             this.excludeSensitiveHeaders = excludeSensitiveHeaders;
+            this.ignoredAuditUsers = ignoredAuditUsers;
             this.ignoredAuditUsersMatcher = WildcardMatcher.from(ignoredAuditUsers);
+            this.ignoredAuditRequests = ignoredAuditRequests;
             this.ignoredAuditRequestsMatcher = WildcardMatcher.from(ignoredAuditRequests);
             this.disabledRestCategories = disabledRestCategories;
             this.disabledTransportCategories = disabledTransportCategories;
         }
 
+        @JsonCreator
+        @VisibleForTesting
+        static Filter from(Map<String, Object> properties) {
+            final boolean isRestApiAuditEnabled = getOrDefault(properties,"enable_rest", true);
+            final boolean isTransportAuditEnabled = getOrDefault(properties,"enable_transport", true);
+            final boolean resolveBulkRequests = getOrDefault(properties, "resolve_bulk_requests", false);
+            final boolean logRequestBody = getOrDefault(properties, "log_request_body", true);
+            final boolean resolveIndices = getOrDefault(properties, "resolve_indices", true);
+            final boolean excludeSensitiveHeaders = getOrDefault(properties, "exclude_sensitive_headers", true);
+            final EnumSet<AuditCategory> disabledRestCategories = AuditCategory.parse(getOrDefault(properties,"disabled_rest_categories", ConfigConstants.OPENDISTRO_SECURITY_AUDIT_DISABLED_CATEGORIES_DEFAULT));
```

Comment:
Think @vrozov had recently modified code for this here - https://github.com/opendistro-for-elasticsearch/security/pull/543/files#diff-c7afdd983cdd9cca05fa869112ff95b9R76-R77

Reply:
It is the same. Look below. 
This function is used by jackson and function at the bottom for elasticsearch yml settings